src/Compilers/Core/Portable/SymbolDisplay/ObjectDisplayOptions.cs

```diff
@@ -40,5 +40,11 @@ internal enum ObjectDisplayOptions
         /// In Visual Basic, replace non-printable characters with calls to ChrW and vb* constants.
         /// </summary>
         EscapeNonPrintableCharacters = 1 << 4,
+
+        /// <summary>
+        /// In C# when specifying <see cref="IncludeCodePoints"/>, use a hexadecimal number for the code point.
+        /// In Visual Basic, use hexadecimal numbers in calls to ChrW.
```

Comment:
miht be worth calling out what happens if this isn't set. (or don't, since it seems reasonably obvious)